config.py

```diff
@@ -233,6 +233,16 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (True, True)
         ),
+        'learning.mozilla.org': (
+            'https://foundation.mozilla.org/en/opportunity/web-literacy/',
+            ReturnCodes.PERMANENT,

```

Comment:
Matthew mentioned this _might_ need a change one day, so let's make this use `ReturnCodes.Temporary`